Manual counting of five functions

apeinot opened the pull request:
Add results in the report
This fix #4 
This fix #5 
This fix #7  
This fix #8  
This fix #9

Samuelhertzberg committed:
Entered the manually counted results.

Entered the manually counted results for the methods 'skip' and 
'genReadOp'. The results were 18 and 22 respectively. Theese number 
should the be bumped by +1. Theese numbers was 
arguably degreed upon by the group members and ultimately decided by 
help from TA.

Samuelhertzberg committed:
Entered the manually counted results.

Entered the manually counted results for the methods 'skip' and 
'genReadOp'. The results were 18 and 22 respectively. Theese number 
should the be bumped by +1. Theese numbers was 
arguably degreed upon by the group members and ultimately decided by 
help from TA.

Samuelhertzberg committed:
Merge branch 'manual_counting' of github.com:apeinot/java into manual_counting

emilelm96 committed:
Adding manual cyclomatic complexity for readNumber() for the report. #7

Filling the table with the value I and Franz came up with it.

apeinot committed:
Added manual count for parse() and updateBindings()

This should fix both #4 and #8

emilelm96 commented:
~It actually fixes #4, #5, #7, #8, #9, since we closed duplicate issue #6 already.~
Fixed.

Samuelhertzberg committed:
Fix counting error

Yesterday we had already added +1 to the number of decision which was 
17. Thus when we did the same today we got 19 which is wrong. #5

emilelm96 committed:
Changed cyclomatic complexity of readNumber(). #7

Emil erroneously counted the default branch in the switch statement, thus rendering the complexity 21 instead of 20. This corresponds with the result generated by lizard, which is good.

francislaus reviewed (approved):
Looks good
